udata_front/theme/gouvfr/assets/js/pages/Admin/Members/Members.vue

```diff
@@ -0,0 +1,274 @@
+<template>
+  <div>
+    <Breadcrumb>
+      <li>
+        <router-link class="fr-breadcrumb__link" to="/">
+          {{ t('Administration') }}
+        </router-link>
+      </li>
+      <li v-if="currentOrganization">
+        <router-link class="fr-breadcrumb__link" to="/">
+          {{ currentOrganization.name }}
+        </router-link>
+      </li>
+      <li>
+      </li>
+    </Breadcrumb>
+    <h1 class="fr-h1 fr-mb-2w">{{ t("Members") }}</h1>
+    <template v-if="membershipRequests.length">
+      <h2 class="subtitle subtitle--uppercase">
+        {{ t("{n} requests", {n: membershipRequests.length}) }}
+      </h2>
+      <MembershipRequest
+        v-for="request in membershipRequests"
+        :key="request.id"
+        :loading="loading"
+        :oid="oid"
+        :request="request"
+        :show-actions="true"
+        @accept="accept"
+        @refuse="refuse"
+      />
+    </template>
+    <h2 class="subtitle subtitle--uppercase fr-mt-2w">
+      {{ t("{n} members", {n: members.length}) }}
+    </h2>
+    <div class="fr-table fr-table--layout-fixed">
+      <table>
+        <thead>
+          <tr>
+            <th scope="col">{{ t("Members") }}</th>
+            <th scope="col">{{ t("Status") }}</th>
+            <th scope="col">{{ t("Action") }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="member in members" :key="member.user.id">
+            <td>{{ member.user.first_name }} {{ member.user.last_name }}</td>
+            <td>
+              <p class="fr-badge">{{ getRoleLabel(member.role) }}</p>
+            </td>
+            <td>
```

Comment:
I think the edit column is also reserved for admin members? Not sure how an editor is supposed to leave an organization though